MediaBrowser.Model/SyncPlay/QueueItem.cs

```diff
@@ -0,0 +1,24 @@
+#nullable disable
+
```

Comment:
```suggestion
```

All new content should have nullable enabled.
I recommend setting values from a constructor.